server/network.js

```diff
@@ -420,3 +454,51 @@ network.listNetwork_ = function(networkPath) {
   });
   return ret;
 };
+
+/**
+ * Find all edges connecting the gene to other exist genes.
```

Comment:
Finds...

The name addGene is a bit confusing. Looks like the main purpose is to find incident edges to existing genes. IncidentEdges may be a better name but there already exists incidentEdges query. Is it possible to combine the two?